api_endpoint.py

```diff
@@ -12,8 +12,8 @@
 from web_utils import bad_request, get_json_params, auth_request, auth_request_get_single_param, auth_request_get_params
 import utils
 import email_utils
-from models import CryptoWithdrawal, FiatDbTransaction, FiatDepositCode, User, UserCreateRequest, UserUpdateEmailRequest, Permission, ApiKey, ApiKeyRequest, BrokerOrder, KycRequest, AddressBook, FiatDeposit, FiatWithdrawal, CryptoAddress, CryptoDeposit, DassetSubaccount
```

Comment:
why is `FiatDepositCode` it is needed in this file

Reply:
its in the master branch and i pulled from the master branch

Reply:
sorry why is `FiatDepositCode` *removed*,  it is needed in this file

I suspect you removed accidentially during a merge conflict

Reply:
readded the FiatDepositCode